components/triggercmd/actions/trigger-command/trigger-command.mjs

```diff
@@ -0,0 +1,48 @@
+import triggercmd from "../../triggercmd.app.mjs";
+import axios from "axios";
+
+export default {
+  name: "Trigger a command",
+  version: "0.0.12",
+  key: "trigger-command",
+  description: "Runs a command on a computer.  Refer to the [TRIGGERcmd Forum](https://triggercmd.com/forum) to learn more.",
+  props: {
+    token: {
+      type: "string",
+      label: "Token from the Instructions page in your account at TRIGGERcmd.com",
+      secret: true
+    },
+    computer: {
+      type: "string",
+      label: "Computer Name",
+    },
+    trigger: {
+      type: "string",
+      label: "Trigger Name",
+    },
+    params: {
+      type: "string",
+      label: "Command parameters",
+      optional: true
+    }
+  },
+  type: "action",
+  methods: {},
+  async run() {
+    // Make an HTTP POST request using axios
+    const resp = await axios({
```

Comment:
Could you please move the request to `app`? Also, it would be great if you use `axios` from `platform/pipedream`. Please take a look on `figma` app for better reference :)

Reply:
Done